sdk/lib/cxxrt/guard.cc

```diff
@@ -1,12 +1,14 @@
 // Copyright Microsoft and CHERIoT Contributors.
 // SPDX-License-Identifier: MIT
 
-#include <cassert>
 #include <cdefs.h>
+#include <debug.hh>
 #include <futex.h>
 #include <limits>
 #include <stdint.h>
 
+using Debug = ConditionalDebug<DEBUG_CXXRT, "cxxrt">;
```

Comment:
I don’t see the xmake option that sets this, so I’m a bit confused about how it compiles at all.

Reply:
It already existed so I didn't have to add it in this PR.